Makefile

```diff
@@ -355,7 +355,7 @@ $(BUILDDIR)/packages.txt:$(GO_TEST_FILES) $(BUILDDIR)
 split-test-packages:$(BUILDDIR)/packages.txt
 	split -d -n l/$(NUM_SPLIT) $< $<.
 test-group-%:split-test-packages
-	cat $(BUILDDIR)/packages.txt.$* | xargs go test -mod=readonly -timeout=15m -race -coverprofile=$(BUILDDIR)/$*.profile.out
+	cat $(BUILDDIR)/packages.txt.$* | xargs go test -mod=readonly -timeout=210s -race -coverprofile=$(BUILDDIR)/$*.profile.out
```

Comment:
why 210s?

Reply:
Nothing takes longer than that, and so by setting it to 210 seconds, we still allow things to pass while catching race conditions fast